airgun/widgets.py

```diff
@@ -2685,6 +2686,34 @@ def read(self):
         return self.column_value.read()
 
 
+class FieldWithEditBtn(Widget):
```

Comment:
```suggestion
class FieldWithEditButton(Widget):
```
I would name it without the abbreviation as we don't use them much in the class names, but your call